src/project/internal/projectactionscontroller.cpp

```diff
@@ -212,6 +213,15 @@ mu::Ret ProjectActionsController::openNotationPageIfNeed()
     return interactive()->open(NOTATION_PAGE_URI).ret;
 }
 
+Ret ProjectActionsController::closeNotationPageIfNeed()
```

Comment:
Better to rename this method to openHomePageIfNeed. Also, its implementation duplicates openNotationPageIfNeed, so you can create a separate method for the implementation and reuse it twice in openNotationPageIfNeed and openHomePageIfNeed

Reply:
I created a new method for opening pages and deleted openHomePageIfNeed and openNotationPageIfNeed. This method accepts a uri as its argument. I hope this is okay